src/networkaccessmanager.h

```diff
@@ -104,6 +104,7 @@ class NetworkAccessManager : public QNetworkAccessManager
     QString m_password;
     QNetworkReply* createRequest(Operation op, const QNetworkRequest& req, QIODevice* outgoingData = 0);
     void handleFinished(QNetworkReply* reply, const QVariant& status, const QVariant& statusText);
+    void _handleNetworkError(QNetworkReply* reply);
```

Comment:
This is a private method, right? So more idiomatic C++ to put it in the `private` section below without the `_`.